lib/src/shared_model/access_key.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:equatable/equatable.dart';
+
+/// Holds properties for generating a Doordash Drive API token
+class AccessKey extends Equatable {
```

Comment:
```suggestion
class TokenProperties extends Equatable {
```
I think when I hear access key, I think of a single key I need to make API calls. Could we rename this to be a bit more clear?

Reply:
I agree with you. When using AccessKey I was following Doordash Drive's doc suggestion. Will apply the change